doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -1,8 +1,31 @@
+```eval_rst
+.. role:: raw-html(raw)
+    :format: html
+```
+
 # Controls and Shortcuts #
 
 The following is a list of input device controls and shortcuts for manipulating the _Node Graph_, _Viewer_, and _Python Editor_.
 
 
+## General ##
+
+```eval_rst
+===================================== =============================================
+Action			                      Control or Shortcut
+===================================== =============================================
+New node graph                        :kbd:`Control` + :kbd:`N`
```

Comment:
@johnhaddon I should have confirmed this before – does the Mac build use <kbd>Control</kbd> or <kbd>⌘</kbd>?

Reply:
Everywhere you would press <kbd>Control</kbd> on Linux, you need to press <kbd>⌘</kbd> on Mac instead.